src/main/java/net/dv8tion/jda/core/entities/User.java

```diff
@@ -109,6 +109,14 @@
      */
     String getEffectiveAvatarUrl();
 
+    /**
+     * The "tag" for this user
+     * This is the equivalent of calling {@link java.lang.String#format(String, Object...) String#format("%#s", {@link net.dv8tion.jda.core.entities.User User})}
```

Comment:
The formatting for this will look like this:

> The "tag" for this user This is the equivalent of calling String#format("%#s", User)

Either add a linefeed or use a period to separate the sentences. Maybe add a note that this is unique for every user.
Two nested link tags are not supported by javadoc and won't render correctly. Besides, I don't think it makes sense to link to the same class in the method docs.

Reply:
If you are writing the javadocs in Intellij, you should be able to just bring up the javadoc viewer from the method (ctrl+q by default) to get an approximate representation of how it will look btw